rllib/agents/dreamer/dreamer.py

```diff
@@ -130,7 +130,11 @@ def sample(self, batch_size: int):
             index = int(random.randint(0, available))
             episodes_buffer.append(episode[index : index + self.length])
 
-        return SampleBatch.concat_samples(episodes_buffer)
+        batch = {}
+        for k in episodes_buffer[0].keys():
+            batch[k] = np.stack([e[k] for e in episodes_buffer], axis=0)
```

Comment:
This sounds like a pretty good fix to me -- Can you test it on some environments, maybe from different libraries, and post a https://tensorboard.dev/ 

I'm fine merging without, as we don't really actively support this algorithm -- at the same time we don't have too many tests for it, which is why I ask if you could do the evaluation that I described above

Reply:
I am a bit busy. I will do the above sometime within a month! If someone else wants to do it asap, feel free!